dashboard/src/app/business/profile/createProfile/createProfile.component.ts

```diff
@@ -369,9 +367,9 @@ export class CreateProfileComponent implements OnInit {
         this.msgs = [];
         this.msgs.push({ severity: 'info', summary: 'Success', detail: 'Form Submitted' });
         this.param.name = value.name;
-        this.param.storageType = value.storageType;
-        this.param.extras.protocol = value.protocol;
-        this.param.extras.policys = value.policys;
+        // this.param.storageType = value.storageType;
+        // this.param.extras.protocol = value.protocol;
+        // this.param.extras.policys = value.policys;
```

Comment:
Why are these commented out?

Reply:
I think these capabilities was moved into ioConnectivityLoS and dataStorageLoS